### PR TITLE
Proposal: added sample for multi database for typeorm app with use v7 API

### PR DESCRIPTION
**Pull request by Tony133**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/nestjs/nest/blob/master/CONTRIBUTING.md
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?
N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

## Other information
This PR Added sample for multi database for typeorm app with use v7 API

### CODE REVIEW

**Comment by Tony133** [comment]
Hi, @BrunnerLivio, what do you think about this example? could there be?

Tony

File: sample/006-multi-db-typeorm-app/src/health/health.controller.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    private health: HealthCheckService,
+    private db: TypeOrmHealthIndicator,
+  ) {}
+
+  @Get()
+  @HealthCheck()
+  check() {
+    return this.health.check([
+      async () => this.db.pingCheck('db1Connection', { connection: getConnection('db1Connection')}),
+      async () => this.db.pingCheck('db2Connection', {connection: getConnection('db2Connection')}),
```
Comment (BrunnerLivio) [inline sample/006-multi-db-typeorm-app/src/health/health.controller.ts:17]: Wouldn't it make more sense to use `@InjectConnection('db2Connection')` in the constructor instead? 

https://github.com/nestjs/typeorm/blob/master/lib/common/typeorm.decorators.ts#L16

**Comment by BrunnerLivio** [comment]
Proposal makes sense for me, thanks!

File: sample/006-multi-db-typeorm-app/src/health/health.controller.ts (hunk 1 above)
Comment (BrunnerLivio) [reply sample/006-multi-db-typeorm-app/src/health/health.controller.ts:17]: @Tony133 I think you've missed this message :)

**Review by Tony133 — COMMENTED** [review]
example updated with @InjectConnection --> https://github.com/nestjs/terminus/pull/933/commits/9a4dd2cd8845ac5067e95abdad2cd9f7e07e80aa

@BrunnerLivio what do you think of this updated example?

File: sample/006-multi-db-typeorm-app/src/health/health.controller.ts (hunk 1 above)
Comment (Tony133) [reply sample/006-multi-db-typeorm-app/src/health/health.controller.ts]: example updated with @InjectConnection --> 9a4dd2c

**Comment by BrunnerLivio** [comment]
Awesome, thank you!

**Pull request merged**